index.html

```diff
@@ -518,11 +518,12 @@ <h2>Algorithms</h2>
       </p>
 
       <p>
-When the RDF Dataset Canonicalization Algorithm [[RDF-CANON]] is used with ECDSA
-algorithms, the cryptographic hashing function that is passed to the algorithm
-MUST be determined by the size of the associated public key. For P-256 keys,
-SHA-2 with 256 bits of output is utilized. For P-384 keys, SHA-2 with 384-bits
-of output is utilized.
+When the RDF Dataset Canonicalization Algorithm (RDFC-1.0) [[RDF-CANON]] is used with ECDSA
+algorithms, the cryptographic hashing function used by RDFC-1.0 MUST depend on the size of 
+the associated public key. For P-256 keys, SHA-2 with 256 bits of output MUST be 
+utilized, which is the default. For P-384 keys, SHA-2 with 384-bits of output MUST be
+utilized, making use of the <a data-cite="RDF-CANON#dfn-hash-algorithm">implementation-dependent setting</a> 
+for RDFC-1.0.
```

Comment:
I was working on this when @dlongley's comment came through... 
```suggestion
When the RDF Dataset Canonicalization Algorithm (RDFC-1.0) [[RDF-CANON]] is used
with ECDSA algorithms, the cryptographic hashing function used by RDFC-1.0 MUST
be chosen based on the size of the associated public key. For P-256 keys, the
default hashing function, SHA-2 with 256 bits of output, MUST be used. For P-384
keys, SHA-2 with 384-bits of output MUST be used, specified via the RDFC-1.0
<a data-cite="RDF-CANON#dfn-hash-algorithm">implementation-specific parameter</a>.
```

Reply:
Thanks @TallTed, I knew I can count on you on this :-) 

I will accept this as as, and ask @dlongley for a re-review to see if he is fine with your version